app/components/Vault/index.js

```diff
@@ -313,11 +314,10 @@ const Vault = (props) => {
 
   const [userInfoYvBoostEth, setUserInfoYvBoostEth] = React.useState(0);
   const [oldPickleGaugeBalance, setOldPickleGaugeBalance] = React.useState(0);
-  const [
-    apyYvBoostEthRecommended,
-    setApyYvBoostEthRecommended,
-  ] = React.useState('NEW ✨');
-  const [vaultAssetsYvBoostEth, setVaultAssetsYvBoostEth] = React.useState(
+  const [apyPickleRecommended, setApyPickleRecommended] = React.useState(
+    'NEW ✨',
```

Comment:
@dudesahn what should fallback state for pickle APY be? Is "New" sufficient? I kind of think we should only show "New" if we haven't shown APY before

Reply:
@dudesahn NEW is fine when it's new, but if it's not new people will know this and likely correct us. I would prefer N/A, even if it looks bad because it's not available and also is not new. Either way I've merged the PR because this should be a non-issue from now on